src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -151,6 +151,12 @@ internal void AddProvider(string workerDir)
                     // Validate workerDescription
                     workerDescription.ApplyDefaultsAndValidate(Directory.GetCurrentDirectory(), _logger);
 
+                    if (workerDescription.IsDisabled == true)
```

Comment:
nit: `if (workerDescription.IsDisabled == true)` -> `if (workerDescription.IsDisabled)`

Reply:
`IsDisabled` is a nullable type property. So `if (workerDescription.IsDisabled)` expression will produce a compilation error (**_Cannot implicitly convert type 'bool?' to 'bool'_**). So we should either do `if (workerDescription.IsDisabled.HasValue && workerDescription.IsDisabled.Value)` or `if (workerDescription.IsDisabled.GetValueOrDefault())`. 

The version I used is a shortcut of the first option (_I felt that is more readable than second_).  `if (IsDisabled == true)` get lowered to `if ((isDisabled.GetValueOrDefault() == true) & isDisabled.HasValue)`

Reply:
Oh, missed that you were working with a nullable boolean here.